tests/functional/object/mcg/test_write_to_bucket.py

```diff
@@ -122,10 +122,24 @@ class TestBucketIO(MCGTest):
                 ],
                 marks=[tier1, on_prem_platform_required],
             ),
+            pytest.param(
+                *[
+                    "CLI",
+                    {
+                        "interface": "CLI",
+                        "namespace_policy_dict": {
+                            "type": "Single",
+                            "namespacestore_dict": {"rgw": [(1, None)]},
+                        },
+                    },
+                ],
+                marks=[tier1, on_prem_platform_required],
```

Comment:
```suggestion
                marks=[tier1, on_prem_platform_required, jira('DFBUGS-747', run=False)],
```

Reply:
Please note I've changed the bug to 'DFBUGS-1035` in an edit (https://issues.redhat.com/browse/DFBUGS-1035). I'm not sure where I got 747 from, but I'm sorry for the mixup.

Reply:
@sagihirshfeld I've added a Jira marker with the correct number.